aliasing/api/combat.py

```diff
@@ -374,3 +427,12 @@ def set_parent(self, parent):
         :type parent: :class:`~aliasing.api.combat.SimpleEffect`
         """
         self._effect.set_parent(parent._effect)
+
+    def __str__(self):
+        return str(self._effect)
+
+    def __repr__(self):
+        return f"<SimpleEffect name=f{self.name!r} duration={self.duration!r} remaining={self.remaining!r}>"
```

Comment:
Is the `f` in `f{self.name!r}` a typo? Format strings are spooky

Reply:
whoops, it is indeed